### PR TITLE
API for LBPOOL Member group

### PR DESCRIPTION
**Pull request by Pushpalatha555**

### CODE REVIEW

File: pkg/client/load_balancer.go [hunk 1]
```diff
@@ -526,3 +526,22 @@ func (lb *LoadBalancerAPIService) GetSpecificLBVirtualServer(
...
+
+func (lb *LoadBalancerAPIService) GetLBPoolMemberGroup(
+	ctx context.Context,
+	serviceID int,
+) (models.GetMemeberGroupForPool, error) {
+	LBMemberGroupResp := models.GetMemeberGroupForPool{}
+	LBMemberGroupInput := &api{
+		method: "GET",
+		path: fmt.Sprintf("%s/%s/%d/%s",
+			consts.NetworksPath, consts.ServerPath, serviceID, consts.GroupsPath),
```
Comment (manjunath-batakurki) [inline pkg/client/load_balancer.go:538]: did you review the reference to old path variable which u used for group ?

**Review by manjunath-batakurki — APPROVED** [review]
LGTM

File: pkg/client/load_balancer.go (hunk 1 above)
Comment (Pushpalatha555) [reply pkg/client/load_balancer.go:538]: yes i used and referred the edge cluster example

Comment (manjunath-batakurki) [reply pkg/client/load_balancer.go:538]: LBPoolMemberGroupPath = "networks/servers/1/groups" should be removed

Comment (Pushpalatha555) [reply pkg/client/load_balancer.go:538]: i created branch from master so i did not get LBPoolMemberGroupPath here

Comment (Pushpalatha555) [reply pkg/client/load_balancer.go:538]: removed in PR https://github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pull/54

**Review by manjunath-batakurki — DISMISSED** [review]
LGTM

**Review by gandharvas — APPROVED** [review]

**Review by gandharvas — APPROVED** [review]

**Review by ravikumar29524 — APPROVED** [review]

**Pull request merged**